src/addon/validation.cpp

```diff
@@ -501,6 +505,10 @@ std::string addon_check_status_desc(unsigned int code)
 			ADDON_CHECK_STATUS::AUTH_TYPE_MISMATCH,
 			N_("The add-on’s forum_auth attribute does not match what was previously uploaded.")
 		},
+		{
+			ADDON_CHECK_STATUS::ICON_TOO_LARGE,
+			N_("The addon’s icon’s file size is too large.")
```

Comment:
I just note that the dominant spelling for 'add-on' in Wesnoth is hyphenated. (You already used this in `client.cpp` so not sure why it's inconsistent here. It's even different from the line above in this file.)

Reply:
Updated.

Reply:
Thanks.